### DISCUSSION TITLE
New metro design

### DISCUSSION THREAD

**Discussion by ghost** [linuxmint/discussions#520] [Ideas]
Latest Linux Mint is really nice, but I'm wondering if Linux Mint could support a variant called Metro that uses a new design.

This is the font (~approx.)

![image](https://github.com/user-attachments/assets/d9a63c4d-23c5-4b5a-bcdb-258beb8f4496)

The design could feature:

- Squarelike
- Gray contrasts
- Expand horizons
- Steel inspirations
- Flowers and all

Like, I know that Linux Mint consists of several different components, but the Mint's GUI component is written in JavaScript, which I do have discomfort with. So I'm wondering how this design could happen.

I'd also add that Linux Mint is great as is, but I really like Metro design too

**Comment by ghost** [comment]
Can you please elaborate? Do you wish to have a secondary theme for Cinnamon that is closer to the Microsoft Metro interface design guidelines? What do you mean by the points such as "steel inspirations" or "flowers and all", do you wish to have the icons changed to various flowers like tulips and carnations?
You might be better off looking at an alternate theme or using KDE's Plasma instead of Cinnamon, as a major overhaul of the appearance of Cinnamon would require some justification.

**Comment by Secret-chest** [comment]
To change the appearance of Cinnamon you need CSS, not JS.
